spirv_glsl.hpp

```diff
@@ -130,16 +138,36 @@ class CompilerGLSL : public Compiler
 			FeatureCount
 		};
 
+		static bool can_feature_be_implemented_wo_extensions(Feature feature)
+		{
+			const static bool retval[FeatureCount]{
+				false,
+				false,
+				false,
+				false,
+				false,
+				false,
+				true, // SubgroupBalloFindLSB_MSB
+				false,
+				true, // SubgroupElect
```

Comment:
yes, but using these features implies using others.

You need another array of what feature implies other features.

Reply:
> yes, but using these features implies using others.

how? no workaround depends on another one

Reply:
Let me give the only example, assuming you will split `SubgroupAll_Any_AllEqual` into `SubgroupAll_Any_AllBool` and `SubgroupAllEqualT`:

## Feature::SubgroupAllEqualT

Requires both:
- SubgroupBrodcast_First
- SubgroupAll_Any_AllBool

Reply:
I foresee that when we implement the rest of the intrinsics (scans and shuffles), we will often need to piggyback off other features for workarounds.

Reply:
Also just got reminded of subgroup elect https://github.com/KhronosGroup/SPIRV-Cross/pull/1476#discussion_r500395226